### ISSUE TITLE
1.2024.9.69: Login popup ups multiple times

### ISSUE THREAD

**Issue by deepakkumar1984** [MscrmTools/XrmToolBox#1369]
The latest version seems very buggy. The login window pops up every time and multiple times when click anything

---
- Connection Controls Version: 1.2024.9.59
- XrmToolBox Version: 1.2024.9.69

**Comment by tao790** [comment]
Yes! I also encountered this problem. I rolled back to the previous version and it worked fine.

**Comment by MscrmTools** [comment]
Can you confirm what kind of connection you are using? Were they created using Microsoft Login Control or not?

**Comment by chironh** [comment]
https://github.com/MscrmTools/XrmToolBox/issues/1363#issuecomment-4342843128
